src/server/wesnothd/game.cpp

```diff
@@ -1425,6 +1428,21 @@ bool game::add_player(player_iterator player, bool observer)
 			user->info().name() + " has the same IP as: " + clones);
 	}
 
+	// If not observer, send the game chat log.
```

Comment:
Why not send to observers?

Reply:
Wasn't sure how chat to observers is handled, it now will be sent to all observers also